frontend/components/global/AutoForm.vue

```diff
@@ -97,20 +97,21 @@
           :label="inputField.label"
           :name="inputField.varName"
           :items="inputField.options"
-          :item-title="inputField.itemText"
-          :item-value="inputField.itemValue"
+          item-title="text"
+          item-value="text"
```

Comment:
We should probably separate these fields into `text` and `value` but I didn't bother since this is used exactly once in the application (here) and is really easy to change if/when we want to.